buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchCluster.java

```diff
@@ -186,6 +186,11 @@ public void plugin(String pluginProjectPath) {
         nodes.all(each -> each.plugin(pluginProjectPath));
     }
 
+    @Override
+    public void upgradePlugin(List<Provider<RegularFile>> plugins) {
+        nodes.all(each -> each.upgradePlugin(plugins));
```

Comment:
I assume the only valid case for this function is during restart upgrades.

Reply:
Yeah I had to override it since it is added in interface `TestClusterConfiguration`.